debian-stuff/katello

```diff
@@ -41,7 +40,7 @@ class pkg_acquire_method:
     __eof = False
 
     def __init__(self):
-        print "100 Capabilities\nVersion: 1.0\nSingle-Instance: true\n\n",
+        sys.stdout.write("100 Capabilities\nVersion: 1.0\nSingle-Instance: true\n\n")
```

Comment:
I would recommend to add following line to the first line of this file:

```python
from __future__ import print_function
```
Then you can use print() method in Python 2 and Python 3 too.

Reply:
Done, thanks